File: update-catalog.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'src/data/quival-catalog.js');

try {
    let content = fs.readFileSync(filePath, 'utf8');

    // Replace product images
    content = content.replace(/\/images\/products\/[^']+/g, '/product-placeholder.svg');

    // Replace placeholder.com
    content = content.replace(/https:\/\/via\.placeholder\.com\/[^']+/g, '/product-placeholder.svg');

    fs.writeFileSync(filePath, content, 'utf8');
    console.log('Successfully updated quival-catalog.js');
} catch (err) {
    console.error('Error updating file:', err);
    process.exit(1);
}

File: update_catalog.py

import re
import os

file_path = '/home/jairoprodev/proyectos/PublicAdis-nextjs/src/data/quival-catalog.js'

with open(file_path, 'r') as f:
    content = f.read()

# Replace placeholder.com
content = content.replace('https://via.placeholder.com/200x80/2563EB/FFFFFF?text=QUIVAL', '/product-placeholder.svg')

# Replace products images
content = re.sub(r'/images/products/[^\'"]+', '/product-placeholder.svg', content)

with open(file_path, 'w') as f:
    f.write(content)

print("Updated catalog successfully")
